upgrading/automated_upgrades.adoc

```diff
@@ -254,7 +254,7 @@ scenarios:
 
```

Comment:
In the NOTE box, is Known Issues a reference to the release notes?

Reply:
There was previously a Known Issues topic in 3.6. I'll remove this NOTE now.